tests/e2e-tests/config/custom-matchers/to-toggle-visibility-of.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { findLabelWithText } from '@woocommerce/blocks-test-utils';
+
+expect.extend( {
+	async toToggleVisibilityOf( toggleText, selector ) {
+		if ( ! selector ) {
+			return {
+				message: () =>
+					`a selector is required to test element's visibility`,
+				pass: false,
+			};
+		}
+		const isElementVisible = async () => !! ( await page.$( selector ) );
```

Comment:
what about display: none, would this cover it? it seems to only cover nonexisting elements.

Reply:
Yeah... Probably wording was not the best here. I refactored this rule so we avoid the word 'visible': 8cd88ba87d036c1f88f21ad1f6a1098409e710c7.